pkg/plugin/canary/fail.go

```diff
@@ -81,31 +80,6 @@ func newCmdFail(streams genericclioptions.IOStreams) *cobra.Command {
 	return cmd
 }
 
-// newCmdReset provides a cobra command wrapping failOptions
```

Comment:
thanks! sorry to be a pain, this should be removed as well https://github.com/DataDog/extendeddaemonset/blob/87217d683f4148d582037db50b579f0796ac881f/README.md#reset-canary-deployment

Reply:
no problem :) 
PR updated